osquery/tables/system/darwin/signature.mm

```diff
@@ -23,6 +22,34 @@
 namespace osquery {
 namespace tables {
 
+// Get the flags to pass to SecStaticCodeCheckValidityWithErrors, depending on
+// the OS version.
+Status getVerifyFlags(SecCSFlags& flags) {
```

Comment:
So first off - thanks for the detailed reply!  I'll be waiting eagerly for that issue to pop up - really interested in your thoughts :smile_cat: